src/math.rs

```diff
@@ -20,3 +20,36 @@ pub fn left_shift64(value: i64, shift: i32) -> i64 {
 pub fn bound<T: Ord + Copy>(min: T, value: T, max: T) -> T {
     max.min(value).max(min)
 }
+
+// Skia cites http://www.machinedlearnings.com/2011/06/fast-approximate-logarithm-exponential.html
+pub fn approx_powf(x: f32, y: f32) -> f32 {
```

Comment:
Your algorithm is scalar, while we want a SIMD one. Otherwise it would be ridiculously slow.

Reply:
There are missing features in `f32x8` (in particular, non-bitwise conversion to and from `u32x8`).